documentation/APIs/core-models.md

```diff
@@ -0,0 +1,581 @@
+  </tr>
+  <tr>
+    <td>Issuing Time</td>
+    <td>int64</td>
+    <td>The time the block is issued.</td>
+  </tr>
+  <tr>
+    <td>Slot Commitment Length</td>
+    <td>uint32</td>
+    <td>The length of serialized Slot Commitment structure.</td>
+  </tr>
+  <tr>
+    <td valign="top">Slot Commitment <code>oneOf</code></td>
+    <td colspan="2">
+      <details open="true">
+        <summary>Slot Commitment</summary>
+        <blockquote>
+          Slot Commitment is an object that contains a summary of a slot. More descriptions are in Slot Commitment section.
+        </blockquote>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Latest Finalized Slot</td>
+    <td>int64</td>
+    <td>The slot index of latest finalized slot.</td>
+  </tr>
+  <tr>
+    <td>Payload Length</td>
+    <td>uint32</td>
+    <td>The length of the following payload in bytes. A length of 0 means no payload will be attached.</td>
+  </tr>
+  <tr>
+    <td valign="top">Payload <code>oneOf</code></td>
+    <td colspan="2">
+      <details open="true">
+        <summary>Generic Payload</summary>
+        <table>
+          <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+          </tr>
+          <tr>
+            <td>Payload Type</td>
+            <td>uint32</td>
+            <td>
+              The type of the payload. It will instruct the node how to parse the fields that follow.
+            </td>
+          </tr>
+          <tr>
+            <td>Payload Bytes</td>
+            <td>ByteArray</td>
+            <td>A sequence of serialized bytes, where the structure depends on `Payload Type`.</td>
+          </tr>
+        </table>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td valign="top">Signature <code>oneOf</code></td>
+    <td colspan="2">
+      <details open="true">
+        <summary>Generic Signature</summary>
+        <table>
+          <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+          </tr>
+          <tr>
+            <td>Signature Type</td>
+            <td>uint8</td>
+            <td>
+              The type of the signature. It will instruct the node how to parse the fields that follow.
+            </td>
+          </tr>
+          <tr>
+            <td>Signature Bytes</td>
+            <td>ByteArray</td>
+            <td>A sequence of serialized bytes, where the structure depends on `Signature Type`.</td>
+          </tr>
+        </table>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Nonce</td>
+    <td>uint64</td>
+    <td>The nonce which lets this block fulfill the PoW requirement.</td>
+  </tr>
+</table>
+
+
+### Block ID
+Block ID denotes an identifier of a block, with type `ArrayBytes[40]`. It is calculated as the following steps, using BLAKE2b-256 hash function:
+
+* `content` is the serialized block **without** signature and nonce.
+* `slot_index` is the slot index of the `Issuing Time` of the block.
```

Comment:
Is that the index in the Slot Commitment struct?

Reply:
No, it's a different one. I'll clarify that in the context!